xbmc/pvr/recordings/PVRRecording.h

```diff
@@ -82,8 +84,23 @@ namespace PVR
     std::string   m_strFanartPath;    /*!< fanart path */
     unsigned      m_iRecordingId;     /*!< id that won't change while xbmc is running */
 
+  private:
```

Comment:
Yeah I have read Meyers too while I was still studying, and I bet that this was relevant in 1995 when compilers didn't optimise this out, but I seriously doubt that it still is, and we use this throughout our code.

Even if it's more efficient after optimisations, we don't allocate a million of these, and not in a time critical thread. This just clutters.